docs/src/pages/[platform]/connected-components/storage/fileuploader/examples/VariationExample.tsx

```diff
@@ -6,12 +6,14 @@ export const VariationExample = () => {
         variation="drop"
         acceptedFileTypes={['image/*']}
         accessLevel="public"
+        // @ts-ignore
         provider="fast" // IGNORE
       />
       <FileUploader
         variation="button"
         acceptedFileTypes={['image/*']}
         accessLevel="public"
+        // @ts-ignore
```

Comment:
Can you try adding `// IGNORE` at the end of these ts-ignore lines? There is a plugin that removes those lines from the rendered code example in the docs

Reply:
ooh good catch!!!